tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,3100 @@
+       "      dtype=&#x27;&lt;U11&#x27;)</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-b1b0031c-8dc4-4d06-81df-c8addeb2ea05' class='xr-section-summary-in' type='checkbox'  ><label for='section-b1b0031c-8dc4-4d06-81df-c8addeb2ea05' class='xr-section-summary' >Indexes: <span>(1)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'><li class='xr-var-item'><div class='xr-index-name'><div>contingency</div></div><div class='xr-index-preview'>PandasIndex</div><div></div><input id='index-97ee2c73-4d81-4282-8239-a38020a8fe1b' class='xr-index-data-in' type='checkbox'/><label for='index-97ee2c73-4d81-4282-8239-a38020a8fe1b' title='Show/Hide index repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-index-data'><pre>PandasIndex(Index([&#x27;tp_count&#x27;, &#x27;tn_count&#x27;, &#x27;fp_count&#x27;, &#x27;fn_count&#x27;, &#x27;total_count&#x27;], dtype=&#x27;object&#x27;, name=&#x27;contingency&#x27;))</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-6239170d-876e-4ce8-acbb-55372e1fb037' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-6239170d-876e-4ce8-acbb-55372e1fb037' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray (contingency: 5)> Size: 40B\n",
+       "array([ 9.,  6.,  2.,  1., 18.])\n",
+       "Coordinates:\n",
+       "  * contingency  (contingency) <U11 220B 'tp_count' 'tn_count' ... 'total_count'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# It is also possible, and briefer, to make the binary contingency table directly from the event operator, as follows:\n",
+    "\n",
+    "contingency_manager = event_operator.make_table(simple_forecast, simple_obs)\n",
```

Comment:
This where you can mention in the tutorial that you can use the `op_fn` arg here and also the `event_threshold_arg`.
E.g.
```python
event_operator.make_table(simple_forecast, simple_obs, event_threshold=1.5, op_fn=operator.lt)
```
I think that this is important to give a simple example for people who want to take the `event_operator.make_table` approach to skip step 1.

Reply:
done